solvers/cyanure.py

```diff
@@ -14,13 +15,16 @@ class Solver(BaseSolver):
 
     def set_objective(self, X, y, lmbd):
         self.X, self.y, self.lmbd = X, y, lmbd
+        if scipy.sparse.issparse(self.X) and \
+           scipy.sparse.isspmatrix_csc(self.X):
```

Comment:
Not a big fan of `\`, I would prefer:
```suggestion
        if (scipy.sparse.issparse(self.X)
                and scipy.sparse.isspmatrix_csc(self.X)):
```